test/plots/projection-null.ts

```diff
@@ -0,0 +1,12 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+import {feature} from "topojson-client";
+
+export async function projectionNull() {
+  const world = await d3.json<any>("data/countries-110m.json");
+  const land = feature(world, world.objects.land);
+  return Plot.plot({
+    projection: null,
```

Comment:
The projection is null by default, so this isn’t needed here, right?

Reply:
The intent was as a memento that in this case the projection is null(ish), but that's already in the test name!

Reply:
I’ll leave it as-is since the geoLine test covers the undefined case.